### DISCUSSION TITLE
How to migrate a local Wordpress website to an online host.

### DISCUSSION THREAD

**Discussion by Anthony1070** [leokhoa/laragon#853] [Laragon's Articles & How-tos]
Hello,

there is probably a guide about what I am asking, but I can't find it, sorry.

I have created a local wordpress website using Laragon. The local website address is "pippo.dev.com". 

I would like to migrate it online, but I can't just backup it locally and restore it online, because the online address il "pippo.com", instead of "pippo.dev.com".

Is there a fast way to solve this issue and put my site online?

Many many thanks in advance.

Cheers,

Antonio

**Comment by SalmanRavoof** [comment]
WordPress sites have two functioning parts:
1. The files
2. The database. 

You can take a backup of those two and import them to your host. But the manual process is a bit tricky, as you need to perform a search and replace to get all the sitewide links updated. 

I suggest you use a plugin like [All In One WP Migration](https://wordpress.org/plugins/all-in-one-wp-migration/) to get the job done. Once you have the backup (also, can be done using the same plugin), you can even pass it on to your host support to set it up (most good hosts will do it gladly).

**Reply by Anthony1070** [reply]
Hello Salman,

thanks for the reply, but sadly it seems that Hostinger, the hosting I generally use, does not offer this service.
They have in fact told me to make the changes by myself, using PHPMYADMIN and a guide.

Since I don't trust my competence in DB handling, what if I rebuild the local site, that is still at its early stages, using the same hostname template as the destination (online) site, that is to say without the "dev" suffix ?

Once finished, I should be able to backup it and restore it online. What do you think, am I right?

Many thanks in advance.

A.

**Reply by SalmanRavoof** [reply]
> Hello Salman,
> 
> thanks for the reply, but sadly it seems that Hostinger, the hosting I generally use, does not offer this service. They have in fact told me to make the changes by myself, using PHPMYADMIN and a guide.
> 
> Since I don't trust my competence in DB handling, what if I rebuild the local site, that is still at its early stages, using the same hostname template as the destination (online) site, that is to say without the "dev" suffix ?
> 
> Once finished, I should be able to backup it and restore it online. What do you think, am I right?
> 
> Many thanks in advance.
> 
> A.

With the plugin I suggested, you don't have to deal with the database. Just set up a plain WordPress site on your host, install the plugin, and then use the backup file from your local site to restore it there. 

There are many plugins that do this. I just suggested one of the many. The previous commenter has also recommended a few more good options.

**Comment by MichaelA2014** [comment]
Just use one of many plugins available online such as backupbuddy (my favorite)  or BackWPup or Duplicator or Updraft. A quick google search returned me dozen of results.
